Apps/Playground/Scripts/experience.js

```diff
@@ -290,6 +291,30 @@ CreateBoxAsync(scene).then(function () {
                         { xrInput: xr.input });
                 }
 
+                // Test image tracking and detection.
+                if (imageTracking) {
+                    const webXRTrackingMeshes = [];
+                    const webXRImageTrackingModule = xr.baseExperience.featuresManager.enableFeature(
+                        BABYLON.WebXRFeatureName.IMAGE_TRACKING,
+                        "latest",
+                        {
+                            images: [
```

Comment:
How do you test this? Do you pull these images up on your dev machine and point your device at it? Can you add some comments?

Reply:
Yes you can pull the image up on your dev machine and test it or you can print the image out and use tracking that way.  I'll add a comment